helpers/db.js

```diff
@@ -30,4 +28,18 @@ async function getAllItems() {
   return items;
 }
 
-export { getAllItems };
+async function getItem(id) {
+  await connectDatabase();
+
+  const item = await Item.findOne({ itemId: id });
+  return item;
+}
+
+async function updateItem(id, item) {
+  await connectDatabase();
+
+  const newItem = await Item.findOneAndUpdate({ itemId: id }, item);
```

Comment:
Be aware that your method does NOT return the updated item here (I hate that too, but thats what is the default)
https://stackoverflow.com/questions/32811510/mongoose-findoneandupdate-doesnt-return-updated-document

Reply:
Thank you very much!